tests/test_smoke.py

```diff
@@ -309,6 +309,22 @@ def test_n_node_job_queue():
     run_one_test(test)
 
 
+def test_large_job_queue():
+    name = _get_cluster_name()
+    test = Test(
+        'job_queue_large',
+        [
+            f'sky launch -y -c {name} --cloud gcp ""',
+            f'for i in `seq 1 75`; do sky exec {name} -d "echo $i; sleep 100000000"; done',
+            f'sky cancel repr 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16',
+            'sleep 20',
+            f'sky queue {name} | grep -v grep | grep RUNNING | wc -l | grep 16',
```

Comment:
Comment on why 16 will be running? We use .5 cpu per task and the default VM has 8 CPUs.

Reply:
Good point! Added. Thanks!